datumaro/plugins/sly_pointcloud_format/extractor.py

```diff
@@ -0,0 +1,187 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from glob import iglob
+import json
+import os.path as osp
+
+from datumaro.components.extractor import (
+    AnnotationType, Cuboid3d, DatasetItem, Importer, LabelCategories,
+    SourceExtractor,
+)
+from datumaro.util.image import find_images
+
+from .format import PointCloudPath
+
+
+class SuperviselyPointcloudExtractor(SourceExtractor):
+    NAME = 'sly_pointcloud'
```

Comment:
```suggestion
class SuperviselyPointCloudExtractor(SourceExtractor):
    NAME = 'sly_point_cloud'
```

"Pointcloud" is not a word, and you consistently spell it as two words in the documentation, so it should be treated as two words in the code, as well.